### ISSUE TITLE
ItemsService not exported from @directus/api package

### ISSUE THREAD

**Issue by elias-summermatter** [directus/directus#19180]
### Describe the Bug

Since version 10.4.0 (also on newest) version import ItemsService  when writing extension does not work:
![image](https://github.com/directus/directus/assets/41843343/07904c7b-c2f8-4e31-9d80-7888e3ce1fb4)
`Error: Module '"@directus/api"' has no exported member 'ItemsService'.ts(2305)`

### To Reproduce

Try to import ItemsService  from  @directus/api  package.

### Directus Version

10.4.0

### Hosting Strategy

Self-Hosted (Custom)

**Comment by rijkvanzanten** [comment]
I don't think those are currently explicitly exported from anywhere

**Comment by Anoesj** [comment]
It really should be though. In a few extensions of mine, I extended the `ItemsService` class. This code is now all broken since the types setup was changed thoroughly a few updates ago. I know you're doing a lot to improve Directus, but the updates lately required some painful migrations..

Note that `ItemsService` (still) isn't even typed at all, which makes DX really bad :disappointed::
```ts
export default defineEndpoint((router, { services }) => {
  const { ItemsService } = services; // 'ItemsService' is not typed here, in fact, `services` is `any`
});
```

**Comment by rijkvanzanten** [comment]
> It really should be though. 

I agree, which is why we started turning these system pieces into individual packages that are meant to be used in extensions

> In a few extensions of mine, I extended the ItemsService class

You can still do this from the services passed in the function type

> This code is now all broken since the types setup was changed thoroughly a few updates ago. I know you're doing a lot to improve Directus, but the updates lately required some painful migrations..

The documented setup on how to use these services hasn't changed. If that were the case, it would've been in the release notes marked as a breaking change. If you're importing directly from distributed files in a way that's not documented nor recommended, things might/can/will break.

> This issue should not have been closed.

Agree to disagree.
